src/contracts/GPv2AllowListAuthentication.sol

```diff
@@ -2,20 +2,24 @@
 pragma solidity ^0.7.6;
 
 import "@gnosis.pm/util-contracts/contracts/StorageAccessible.sol";
+import "@openzeppelin/contracts/proxy/Initializable.sol";
 import "@openzeppelin/contracts/utils/EnumerableSet.sol";
 import "./interfaces/GPv2Authentication.sol";
 
 /// @title Gnosis Protocol v2 Access Control Contract
 /// @author Gnosis Developers
-contract GPv2AllowListAuthentication is GPv2Authentication, StorageAccessible {
+contract GPv2AllowListAuthentication is
+    GPv2Authentication,
+    Initializable,
+    StorageAccessible
+{
     using EnumerableSet for EnumerableSet.AddressSet;
 
     address public manager;
 
     EnumerableSet.AddressSet private solvers;
 
-    function initializeManager(address manager_) external {
-        require(manager == address(0), "GPv2: already initialized");
+    function initializeManager(address manager_) external initializer {
```

Comment:
Looks like you're putting back the initializer.

Reply:
Yes, as I said before, the issue for me was adding a dependency to `@openzepping/upgrade-contracts` for the `Initializable` when we could just use the one from `@openzeppelin/contracts`: https://github.com/gnosis/gp-v2-contracts/pull/351#discussion_r568532290